app/assets/stylesheets/forever_style_guide/globals/_colors.scss

```diff
@@ -7,13 +7,13 @@
 $color-forever_green:       colorify('green', #7dc142) !default;
 $color-forever_spare:       colorify('spare', #7ec24d) !default;
 $color-forever_historian:   colorify('historian', #67bd68) !default;
-$color-forever_valet:       colorify('valet', #50bd94) !default;
 $color-forever_services:    colorify('services', #39b3be) !default;
 $color-forever_artisan:     colorify('artisan', #28a6de) !default;
 $color-forever_blue:        colorify('blue', #2b91cf) !default;
 $color-forever_ambassador:  colorify('ambassador', #a4314c) !default;
 $color-forever_p2p:         colorify('p2p', #f89406) !default;
 
+$color-forever_emerald:     colorify('emerald', #50bd94) !default;
```

Comment:
I don't think we should be changing our colors at this time because it touches so much

I understand using a product that is no longer represented on the site as a naming convention is going to result in a to do but I'd rather take time to learn where the business sits in terms of a path forward before making this drastic of an update